Dockerfile

```diff
@@ -22,10 +20,12 @@ ADD requirements.txt /code/requirements.txt
 ADD package.json /code/package.json
 ADD app/assets /code/app/assets
 
-RUN pip3 install -U -I -r /code/requirements.txt
+RUN git clone git://github.com/yyuu/pyenv.git .pyenv
+ENV PYENV_ROOT /code/.pyenv
+ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
 
 ADD . /code
 
-RUN yarn compile
+RUN pyenv install &&  pyenv rehash && pip install -U pip setuptools wheel
 
-ENTRYPOINT python3 application.py runserver
+ENTRYPOINT pip install -r requirements.txt && yarn compile && python application.py runserver
```

Comment:
We're currently using Docker for dev only; which is why the container imports the current directory into /code (so you get live filesystem updates). I think if we wanted to use Docker for builds we should re-think this.

Cryptography is failing because of the last merge, need to fix that. Because we aren't using ./scripts/run_app.sh it isn't installing the Git version of that package. We should discuss that as well.